File: sdks/ruby/lib/cyberarmor/providers/perplexity.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class Perplexity < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'sonar')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'perplexity', model: model)
      end
    end
  end
end

File: sdks/php/src/Providers/Perplexity.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class Perplexity extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'sonar')
    {
        parent::__construct($client, $openai, $tenantId, 'perplexity', $model);
    }
}

